Views/Shared/Components/Category/Default.cshtml

```diff
@@ -0,0 +1,6 @@
+﻿@model IEnumerable<Category>
+
+@foreach (var category in Model)
+{
+        <li><a asp-area="" asp-controller="Category" asp-action=@category.Id>@category.Title</a></li>
```

Comment:
Kam reikalingi du viewsai kategorijimos ? Gal galima šitą padaryti partialView ir pernaudoti aname view'se kur naudojamos kategorijos?

Reply:
Šitas view priklauso CategoryViewComponent'ui ir yra naudojamas meniu elementų atvaizdavimui.

Reply:
Šitas view priklauso CategoryViewComponent'ui ir yra naudojamas kiekviename svetainės puslapyje matomo meniu elementų atvaizdavimui.

Reply:
Šitas view priklauso CategoryViewComponent'ui ir yra naudojamas kiekviename svetainės puslapyje matomo meniu atvaizdavimui.